src/osd/PG.cc

```diff
@@ -2743,13 +2743,12 @@ bool PG::_has_removal_flag(ObjectStore *store,
 }
 
 int PG::peek_map_epoch(ObjectStore *store,
-		       spg_t pgid,
+		       coll_t& coll,
```

Comment:
Nov 2014 as part of 6074bdd155d1ea845070c6727f3dd24a16580fb6 Sage changed this from coll_t to spg_t.  Is there a reason to change it back again?